### PR TITLE
docs(replays): add rage/dead selector as searchable property

### PR DESCRIPTION
**Pull request by michellewzhang**
Closes https://github.com/getsentry/team-replay/issues/206

### CODE REVIEW

File: src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx [hunk 1]
```diff
@@ -120,6 +120,12 @@ The number of URLs that the user visited during a replay recording.
 
 - **Type:** number
 
+### `dead.selector`
+
+Similar to the `click.selector` search property, but only queries on dead clicks. An element identified using a subset of CSS selector syntax. For example, `span#section-1` or `span.active` or `span[role=button]` or `span#section-1.active[role=button]` would all match the element `<span id="section-1" class="active" role="button"/>`. Note that, CSS combinators, pseudo selectors, and attr selectors other than `=` are not supported.
```
Comment (ryan953) [inline src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx:125]: is the node type, `span`, required?

Comment (michellewzhang) [reply src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx:125]: SCR-20230928-loxa
doesn't seem like it

Comment (billyvg) [reply src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx:125]: Maybe give a mix of suggestions to give user an idea that you *can* use the el type

> `#section-1` or `span.active` or `span[role=button]` or `.active[role=button]`

**Review by billyvg — COMMENTED** [review]
💀 .selector

File: src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx (hunk 1 above)
Comment (ryan953) [reply src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx]: Ya, more mixed selectors is what i was thinking. I should've just said that, don't mean to be passive aggressive sounding

File: src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx [hunk 2]
```diff
@@ -120,6 +120,12 @@ The number of URLs that the user visited during a replay recording.
 
 - **Type:** number
 
+### `dead.selector`
+
+Similar to the `click.selector` search property, but only queries on dead clicks. An element identified using a subset of CSS selector syntax. For example, `#section-1` or `span.active` or `span[role=button]` or `.active[role=button]` would all match the element `<span id="section-1" class="active" role="button"/>`. Note that, CSS combinators, pseudo selectors, and attr selectors other than `=` are not supported.
```
Comment (ryan953) [inline src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx:125]: Should we define or link the `dead clicks` in the first sentence?

**Review by ryan953 — APPROVED** [review]

File: src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx (hunk 2 above)
Comment (michellewzhang) [reply src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx]: Added a link to where we define dead/rage clicks in the docs already

Comment (ryan953) [reply src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx]: woo! link right to the text too!

**Pull request merged**